documentation/operators/src/nodes/maintenance.md

```diff
@@ -0,0 +1,699 @@
+- [x] Include how to move a mix node from one VPS to another while keeping the delegation etc (ie init a new mix node and move `/home/nym/.nym/mixnodes/data/*` there ?)
+    - copy that dir over to the new vps
+    - re-run init - remember that init doesn't overwrite existing keys - to generate a  config with the new listening address etc
+    - change the node info via the wallet
+    - Since otherwise their keys will point to the old IP address in the smart contract, and their node will (a) not be able to be connected to, and (b) fail uptime checks 
+--->
+## Useful commands
+
+> Adding `--no-banner` startup flag will prevent Nym banner being printed even if run in tty environment.
+
+**build-info**
+
+A `build-info` command prints the build information like commit hash, rust version, binary version just like what command `--version` does. However, you can also specify an `--output=json` flag that will format the whole output as a json, making it an order of magnitude easier to parse.
+
+For example `./target/debug/nym-network-requester --no-banner build-info --output json` will return:
+
+```
+{"binary_name":"nym-network-requester","build_timestamp":"2023-07-24T15:38:37.00657Z","build_version":"1.1.23","commit_sha":"c70149400206dce24cf20babb1e64f22202672dd","commit_timestamp":"2023-07-24T14:45:45Z","commit_branch":"feature/simplify-cli-parsing","rustc_version":"1.71.0","rustc_channel":"stable","cargo_profile":"debug"}
+```
+
+## Upgrading your node
+
+> The process is the similar for mixnode, gateway and network requester. In the following steps we use a placeholder `<NODE>` in the commands, please change it for the type of node you want to upgrade. Any particularities for the given type of node are included.
+
+Upgrading your node is a two-step process:
+* Updating the binary and `~/.nym/<NODE>/<YOUR_ID>/config.toml` on your VPS
+* Updating the node information in the [mixnet smart contract](https://nymtech.net/docs/nyx/mixnet-contract.html). **This is the information that is present on the [mixnet explorer](https://explorer.nymtech.net)**.
+
+### Step 1: Upgrading your binary
+Follow these steps to upgrade your mix node binary and update its config file:
+* pause your mix node process.
+* replace the existing binary with the newest binary (which you can either [compile yourself](https://nymtech.net/docs/binaries/building-nym.html) or grab from our [releases page](https://github.com/nymtech/nym/releases)).
+* re-run `init` with the same values as you used initially. **This will just update the config file, it will not overwrite existing keys**.
+* restart your mix node process with the new binary.
+
+> In case of a network requester this is all all, the following step is only for mix nodes and gateways.
+
+### Step 2: Updating your node information in the smart contract
```

Comment:
this is a little confusing given that above you mention this upgrade info also works for the network requester binary - right now there is no smart contract info about network requesters

Reply:
I think make it clear that these instructions are for mix nodes and gateways, and then have the separate section below for NRs